### PR TITLE
Release ip messaging

### PR DESCRIPTION
**Pull request by Ragil**

### CODE REVIEW

File: lib/twilio-ruby/rest/ip-messaging/credentials.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+          @path, @client = path, client
+          @submodule = :IpMessaging
+        end
+      end
+
+      class Credential < InstanceResource
+        def initialize(path, client, params={})
+          super
+          @submodule = :IpMessaging
+          resource :channels,
```
Comment (jingming) [inline lib/twilio-ruby/rest/ip-messaging/credentials.rb]: copy/pasta error?

Comment (Ragil) [inline lib/twilio-ruby/rest/ip-messaging/credentials.rb]: I also made the changes for edge

https://github.com/twilio/ruby-private/pull/37

**Comment by Ragil** [comment]
@jingming review plz

**Comment by jingming** [comment]
+1

**Pull request merged**